main/main.zkasm

```diff
@@ -106,11 +106,15 @@ skipSetGlobalExitRoot:
 txLoopRLP:
         $ => A          :MLOAD(lastCtxUsed)
         A+1 => CTX      :MSTORE(lastCtxUsed)
+        ; set flag isLoadRLP to 1
+        1               :MSTORE(isLoadRLP)
 
         $ => A          :MLOAD(batchL2DataLength)
         $ => C          :MLOAD(batchL2DataParsed)
         C - A           :JMPN(loadTx_rlp, endCheckRLP)
 endCheckRLP:
+        ; set flag isLoadRLP to 0
+        0               :MSTORE(isLoadRLP)
```

Comment:
The var now is of type CTX. We have to options:
- Set the flag to 0 before calling `loadTx_rlp`: this way, all the CTX are set to 0
- Make the var GLOBAL: in this case we should set it to 1 only once just before `txLoopRLP`
The current situation is that we are only setting the flag to 0 for the last processed context